reco_utils/evaluation/spark_diversity_evaluation.py

```diff
@@ -0,0 +1,373 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from pyspark.sql.types import *
+from pyspark.sql import functions as F
+
+from reco_utils.common.constants import (
+    DEFAULT_USER_COL,
+    DEFAULT_ITEM_COL,
+)
+
+
+class DiversityEvaluation:
+    def __init__(
```

Comment:
You should move the general information for the class here. Otherwise Sphinx won't show anything for this class.

Reply:
addressed.